src/Dropdown/Dropdown.test.jsx

```diff
@@ -36,6 +36,12 @@ describe('<Dropdown />', () => {
     it('with menu closed', () => {
       menuOpen(false, wrapper);
     });
+
+    it('no focus on just update', () => {
+      expect(document.body.outerHTML).toEqual(document.activeElement.outerHTML);
```

Comment:
Can we simplify it like this
```
const activeElementId = document.activeElement.id;
wrapper.instance().forceUpdate();
expect(activeElementId).toEqual(document.activeElement.id);
```

Reply:
`document.activeElement.id` returns `""` so, that will not verify correctly.
Instead we can write this:
`     const activeElementHtml = document.activeElement.outerHTML;
      wrapper.instance().forceUpdate();
      expect(activeElementHtml).toEqual(document.activeElement.outerHTML);
`